doc/classes/SpatialMaterial.xml

```diff
@@ -117,6 +117,7 @@
 		</member>
 		<member name="depth_deep_parallax" type="bool" setter="set_depth_deep_parallax" getter="is_depth_deep_parallax_enabled">
 			If [code]true[/code], the shader will read depth texture at multiple points along the view ray to determine occlusion and parrallax. This can be very performance demanding, but results in more realistic looking depth mapping.
+			[b]Note:[/b] On mobile platforms, deep parallax mapping may exhibit visible artifacts unless [ProjectSettings]' [code]rendering/gles2/compatibility/enable_high_float.Android[/code] is [code]true[/code]. However, enabling high-precision floating-point computations has a performance cost and is not compatible with all devices.
```

Comment:
Since the project setting in question is only defined for the `Android` feature tag, I can't use a direct reference as `makerst.py` otherwises complains and crashes:

```
Checking for errors in the XML class reference...
ERROR: Bad reference: 'ProjectSettings.rendering/gles2/compatibility/enable_high_float.Android', file: SpatialMaterial
Traceback (most recent call last):
  File "/home/hugo/Documents/Git/godotengine/godot/doc/tools/makerst.py", line 1046, in <module>
    main()
    └ <function main at 0x7f94c624af70>
  File "/home/hugo/Documents/Git/godotengine/godot/doc/tools/makerst.py", line 357, in main
    make_rst_class(class_def, state, args.dry_run, args.output)
    │              │          │      │             └ Namespace(path=['doc/classes', 'modules'], filter='', output='.', dry_run=True)
    │              │          │      └ Namespace(path=['doc/classes', 'modules'], filter='', output='.', dry_run=True)
    │              │          └ <__main__.State object at 0x7f94c6251040>
    │              └ <__main__.ClassDef object at 0x7f94c44609a0>
    └ <function make_rst_class at 0x7f94c6182040>
  File "/home/hugo/Documents/Git/godotengine/godot/doc/tools/makerst.py", line 552, in make_rst_class
    f.write(rstize_text(property_def.text.strip(), state) + "\n\n")
    │       │           │                          └ <__main__.State object at 0x7f94c6251040>
    │       │           └ <__main__.PropertyDef object at 0x7f94c4461c70>
    │       └ <function rstize_text at 0x7f94c6182160>
    └ <_io.TextIOWrapper name='/dev/null' mode='w' encoding='utf-8'>
  File "/home/hugo/Documents/Git/godotengine/godot/doc/tools/makerst.py", line 740, in rstize_text
    class_param, method_param = ss
                                └ ['ProjectSettings', 'rendering/gles2/compatibility/enable_high_float', 'Android']
ValueError: too many values to unpack (expected 2)
```